### PR TITLE
QPPA-3668: Added cost measures for PY2020

### PR DESCRIPTION
**Pull request by bartcone**
#### Motivation for change

Cost measures were provided for PY2020.

#### What is being changed

Cost measures have been added to the overall `measures-data.json`. Updated the 2020 `measures-schema.yaml` to allow for `null` values in `overallAlgorithm` property. `build-2020-measures` run script now updated to process cost measures.

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [ ] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPA-3668

### CODE REVIEW

File: measures/2020/measures-data.json [hunk 1]
```diff
@@ -9517,5 +9517,345 @@
...
+    "metricType": "costScore",
+    "firstPerformanceYear": 2017,
+    "lastPerformanceYear": null,
+    "isInverse": true,
+    "overallAlgorithm": "simpleAverage",
+    "submissionMethods": [
+      "administrativeClaims"
+    ],
+    "measureSpecification": {
+      "measureInformation": "https://qpp.cms.gov/docs/cost_specifications/2019-12-17-mif-tpcc.pdf"
```
Comment (joshboley) [inline measures/2020/measures-data.json]: One change.  We determined we were going to use `default`, not `measureInformation`.

Comment (bartcone) [reply measures/2020/measures-data.json]: For cost measures? We used `default` for `PI` in #315.

Comment (joshboley) [reply measures/2020/measures-data.json]: Yes.  As a pattern, if the measure spec is not associated with a `submissionMethod` (which `measureInformation` is not), then the key will be `default`

Comment (bartcone) [reply measures/2020/measures-data.json]: @joshboley addressed feedback.

**Pull request merged**